app/scripts/services/globalPromotionService.ts

```diff
@@ -83,7 +84,24 @@ export class GlobalPromotionService {
     this.globalPromotionsCache.put(cacheKey, [...cachedData, newPromotion]);
   }
 
-  hasPromotionsForConference(
+  hasPromotionsForConference(conference: Conference): boolean {
+    const hasRegistrantTypeAllowingGlobal = conference.registrantTypes.some(
+      (registrantType) => registrantType.eligibleForGlobalPromotions,
+    );
+
+    const hasGlobalPromotions =
+      hasRegistrantTypeAllowingGlobal &&
+      conference.ministry &&
```

Comment:
Just noticed that the `ministry` field on `Conference` should be nullable. Can you tweak that real quick?